src/test/java/com/example/document/controller/DocumentControllerTest.java

```diff
@@ -295,51 +244,27 @@ void deleteEditDocumentEntry() throws Exception {
         assertThat(actualDocumentEntryDTOList)
                 .hasSize(1)
                 .usingRecursiveFieldByFieldElementComparatorIgnoringFields("id")
-                .doesNotContain(documentEntryDTO1);
+                .doesNotContain(documentEntryDTOS.get(0));
 
         DocumentEntryDTO actualDocumentEntryDTO = actualDocumentEntryDTOList.stream().findFirst().orElseThrow();
 
-        assertThat(actualDocumentEntryDTO).usingRecursiveComparison().ignoringFields("id").isEqualTo(documentEntryDTO2);
+        assertThat(actualDocumentEntryDTO).usingRecursiveComparison().ignoringFields("id").isEqualTo(documentEntryDTOS.get(1));
     }
 
-
     @Test
     void deleteMultipleDocumentEntries() throws Exception {
-        DocumentEntryDTO documentEntryDTO1 = DocumentEntryDTO.builder()
-                .uploadDate(LocalDate.of(2022, 10, 13))
-                .effectiveDate(LocalDate.of(2022, 9, 30))
-                .employeeId(1L)
-                .benefitId(2L)
-                .amount(BigDecimal.valueOf(300))
-                .build();
-        DocumentEntryDTO documentEntryDTO2 = DocumentEntryDTO.builder()
-                .uploadDate(LocalDate.of(2022, 10, 13))
-                .effectiveDate(LocalDate.of(2022, 9, 30))
-                .employeeId(2L)
-                .benefitId(2L)
-                .amount(BigDecimal.valueOf(400))
-                .build();
-        DocumentEntryDTO documentEntryDTO3 = DocumentEntryDTO.builder()
-                .uploadDate(LocalDate.of(2022, 10, 13))
-                .effectiveDate(LocalDate.of(2022, 9, 30))
-                .employeeId(2L)
-                .benefitId(2L)
-                .amount(BigDecimal.valueOf(500))
-                .build();
-
         List<DocumentEntryDTO> documentEntryDTOS = new ArrayList<>();
-        documentEntryDTOS.add(documentEntryDTO1);
-        documentEntryDTOS.add(documentEntryDTO2);
-        documentEntryDTOS.add(documentEntryDTO3);
+        IntStream.range(0, 5).forEach(i -> documentEntryDTOS.add(createDocumentEntryDTO(i)));
 
-        Long firstId = documentService.insertDocumentEntries(documentEntryDTOS).stream().findFirst().get().getId();
-        Long secondId = firstId + 1L;
+        documentService.insertDocumentEntries(documentEntryDTOS);
 
-        String idFirst = "" + firstId;
-        String idSecond = "" + secondId;
+        List<String> ids = documentService.getAllDocuments().stream()
+                .map(x -> String.valueOf(x.getId()))
```

Comment:
What is `x` here?
I suggest to use more readable name like `documentEntryDto` or something like that